pkg/systemd/quadlet/quadlet.go

```diff
@@ -486,6 +488,11 @@ func ConvertContainer(container *parser.UnitFile, names map[string]string, isUse
 		podman.addf("--cap-add=%s", strings.ToLower(caps))
 	}
 
+	shmSize, hasShmSize := container.Lookup(ContainerGroup, KeyShmSize)
```

Comment:
The doc says that the size may be omitted, but I'm not sure this takes it into account (or at least, there's not test that checks it)

Reply:
I think that's an internal Podman behavior that would need a runtime test.